src/recurcive.ts

```diff
@@ -0,0 +1,15 @@
+function binarySearch(array: number[], req: number): any {
+  const midPoint = Math.floor(array.length / 2);
+
+  if (array[midPoint] === req) return true; //base case
+
+  if (array[midPoint] < req && array.length > 1)
```

Comment:
```suggestion
  if (array[midPoint] < req && array.length >= 1)
```
Para verificar que el arreglo no sea vacío